css/main.css

```diff
@@ -1,3 +1,163 @@
-/* Add your custom CSS here */
+* {
+    font-family: 'Merriweather Sans', sans-serif;
+    margin: 0;
+    padding: 0;
+    line-height: 1.4rem;
 
-/* Dont' forget to link this file to your HTML in the <head> */
+}
+
+.logo {
+    height: 5rem;
+    width: 5rem;
+    padding: 1rem;
+}
+
+header {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: absolute;
+}
+
+ul {
+    padding: 1rem;
+}
+
+ul>li {
+    list-style: none;
+    display: inline-flex;
+}
+
+li>a {
+    text-decoration: none;
+    color: black;
+    padding-right: 1.5rem;
+}
+
+.hero {
+    /* text-align: center; */
+    background-image: url("../css/img/hero-house.jpeg");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    min-height: 80vh;
+
+}
+
+.hero>h2 {
+    padding-top: 150px;
+    text-align: center;
+
+}
+
+div>img {
+    max-width: 100%;
+}
+
+#houses>h2 {
+    text-align: center;
+    padding: 1rem;
+}
+
+.second-hero-section {
+    background-image: url("https://images.pexels.com/photos/280221/pexels-photo-280221.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500");
+    margin-top: 1rem;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    min-height: 25vh;
+}
+
+.about>h2,
+p {
+    text-align: center;
+
+}
+
+.about>h2 {
+    padding: 1rem;
+}
+
+.about>p {
+    text-align: justify;
+}
+
+.third-hero-section {
+    background-image: url("https://images.pexels.com/photos/1438832/pexels-photo-1438832.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500");
+    margin-top: 1rem;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    min-height: 25vh;
```

Comment:
Given that the `.second-hero-section` and `.third-hero-section` share a lot of styles, could you think of a way to optimise this?